### ISSUE TITLE
not loading sinthesizer driver SAPI5 in NVDA 2018 version

### ISSUE THREAD

**Issue by Manjit-123** [nvaccess/nvda#8646]
I want to use microsoft sapi5 Version 5  on my nvda 2018 but when ever I am trying to load this message is appearing every time. "cannot load the SAPI 5 sinthesizer" please solve my problem. here is my log
[nvda.log](https://github.com/nvaccess/nvda/files/2299609/nvda.log)

**Comment by Mohamed00** [comment]
Try resetting your configuration to factory defaults, it seems that the voice you selected was uninstalled. Have you installed and uninstalled language packs recently?

**Comment by Manjit-123** [comment]
I configured it to factory default. I have uninstall and reinstalled some akapela voices add-on. but gave no harm to SAPI5 sinth drivers. my required voice is  also installed with no extra effort and working on jaws well. @mahammad00

**Comment by Brian1Gaff** [comment]
I believe that if its a Jaws voice download it will not run in nvda.Their 
licence is exclusive. Same seems to go for vocaliser voices from what I can 
tell. I can quite clearly see the installation but only Jaws can see it as a 
choice.
I wonder if its the same for sapi five?
 My sapi five are directly licenced from the supplier through dolphin in my 
case.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
----- Original Message -----

**Comment by netblue44** [comment]
Sum tymes sappie 5 is broken you will have to reinstall zappie 5 
download it from it and try aggain

**Comment by Adriani90** [comment]
@Manjit- are you still having this issue in NVDA 2019.1.1?

**Comment by Adriani90** [comment]
We do not have an update from the initial author since more than 1.5 years. I am closing this as abandoned. If you still have this issue please comment on it and we can reopen it. Thanks.
